Providers/Profile.swift

```diff
@@ -24,13 +21,26 @@ class ProfileFileAccessor : FileAccessor {
                 return nil
             }
         }
-
         return path
     }
 
-    func move(src: String, dest: String) -> Bool {
-        if let f = get(src) {
-            if let f2 = get(dest) {
+    func getDir(name: String?, basePath: String?) -> String? {
```

Comment:
Sorry, its taken me a bit to come back to this. I tried to do this initially, but Swift doesn't allow default parameter values in protocols. There's no way to do this that I know of....